src/cascadia/TerminalSettingsEditor/LaunchViewModel.cpp

```diff
@@ -23,6 +27,130 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         _LaunchModeList.RemoveAt(6); // fullscreenFocus
         _LaunchModeList.RemoveAt(3); // maximizedFullscreen
         INITIALIZE_BINDABLE_ENUM_SETTING(WindowingBehavior, WindowingMode, WindowingMode, L"Globals_WindowingBehavior", L"Content");
+
+        // Add a property changed handler to our own property changed event.
+        // This propagates changes from the settings model to anybody listening to our
+        // unique view model members.
+        PropertyChanged([this](auto&&, const PropertyChangedEventArgs& args) {
+            const auto viewModelProperty{ args.PropertyName() };
+            if (viewModelProperty == L"CenterOnLaunch")
+            {
+                _NotifyChanges(L"LaunchParametersCurrentValue");
+            }
+        });
+    }
+
+    winrt::hstring LaunchViewModel::LaunchParametersCurrentValue()
+    {
+        // todo: get all the current values and combine them into a string
+        const auto launchModeString = CurrentLaunchMode().as<EnumEntry>()->EnumName();
+        const auto centerOnLaunchString = CenterOnLaunch() ? RS_(L"Globals_CenterOnLaunchOn") : RS_(L"Globals_CenterOnLaunchOff");
+
+        winrt::hstring result;
+
+        if (UseDefaultLaunchPosition())
+        {
+            result = fmt::format(L"{}, {}, {}", launchModeString, RS_(L"Globals_LaunchModeDefault/Content"), centerOnLaunchString);

```

Comment:
I would have structured it a bit differently.

It would say:

```
Maximized, Default Position

Normal, Default Position, Centered

Minimized, (3, 3)

Focus, (4, 100), Centered
```

That way, we wouldn't include the less-meaningful strings "on" and "off" and we _would_ only show contextually-appropriate data.

Reply:
(That is: replace `off` with nothing, and `on` with `Centered`)